priam/src/main/java/com/netflix/priam/defaultimpl/StandardTuner.java

```diff
@@ -213,7 +213,7 @@ private void writeCassandraSnitchProperties() {
         if (Strings.isNullOrEmpty(dcSuffix)) {
             properties.remove("dc_suffix");
         } else {
-            properties.put("dc_suffix", "_" + dcSuffix);
+            properties.put("dc_suffix", dcSuffix);
```

Comment:
question - on line 212, do you think its worth doing a check on the 1st character of dcsuffix to be a special character and not a numeric/alphabet character? may be that helps to avoid creating rings with wrong names in case someone forgets to add a "_" on the ops side.